patches/chromium/fix_add_support_for_webauthn_on_macos.patch

```diff
@@ -0,0 +1,315 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Eric Rahm <ericrahm@gmail.com>
+Date: Mon, 22 Nov 2021 12:49:32 -0800
+Subject: fix: Add support for WebAuthn on macOS
+
+Adds support for WebAuthn by including existing Chromium support and modifying
+parts that rely on Chromium specific features.
+
+diff --git a/chrome/browser/ui/views/chrome_typography.cc b/chrome/browser/ui/views/chrome_typography.cc
+index 2383687aee31ceaa961e8b76898851d5fbf6567d..2175d1f6b8006b2d820f6819b56156234ed9a8e0 100644
+--- a/chrome/browser/ui/views/chrome_typography.cc
++++ b/chrome/browser/ui/views/chrome_typography.cc
+@@ -5,7 +5,6 @@
+ #include "chrome/browser/ui/views/chrome_typography.h"
+ 
+ #include "build/build_config.h"
+-#include "chrome/browser/ui/views/location_bar/location_bar_view.h"
+ #include "ui/base/default_style.h"
+ #include "ui/base/pointer/touch_ui_controller.h"
+ #include "ui/base/resource/resource_bundle.h"
+@@ -75,7 +74,8 @@ void ApplyCommonFontStyles(int context,
+     case CONTEXT_OMNIBOX_DEEMPHASIZED: {
+       const int omnibox_primary_delta =
+           GetFontSizeDeltaBoundedByAvailableHeight(
+-              LocationBarView::GetAvailableTextHeight(),
++              // LocationBarView::GetAvailableTextHeight(),
```

Comment:
I feel like the positioning of the dialog should probably be customizable by apps? Can we get some screenshots of how this looks?

Reply:
I believe that's the point of `marginTop`?

![image](https://user-images.githubusercontent.com/426919/150265606-98d765e9-87f4-49fd-9f80-ca328837aaf5.png)

![image](https://user-images.githubusercontent.com/426919/150265652-b25801af-f633-4e2e-a098-6aa820999128.png)

Reply:
@nornagon Would you be okay with focusing on the core functionality in this PR and iterating on further customization in a follow up?

Reply:
This is with:
```TypeScript
mainWindow.setModalDialogOptions({ marginTop: 100 });
```
![image](https://user-images.githubusercontent.com/426919/150266686-329bf7b9-d3a6-4f15-840a-4064db5f6a01.png)